components/climber.py

```diff
@@ -70,6 +75,15 @@ def enable_soft_limits(self) -> None:
             CANSparkMax.SoftLimitDirection.kReverse, True
         )
 
+    def is_real_climb(self) -> bool:
+        # Climbs in the last 20 seconds are real climbs...
+        return (
+            wpilib.DriverStation.getMatchTime() < 20 and self.seen_deploy_limit_switch
+        )
+
+    def should_lock_mechanisms(self) -> bool:
+        return self.is_real_climb() or not self.has_climb_finished()
```

Comment:
maybe we want to change this line to `return self.is_real_climb() and self.has_climb_finished()`
and remove the self.seen_deploy_limit_switch variable